docs/docs/administration/oauth.md

```diff
@@ -8,7 +8,7 @@ Unable to set `app.immich:/` as a valid redirect URI? See [Mobile Redirect URI](
 
 ## Overview
 
-Immich supports 3rd party authentication via [OpenID Connect][oidc] (OIDC), an identity layer built on top of OAuth2. OIDC is supported by most identity providers, including:
+Immich supports 3rd party authentication via [OpenID Connect][oidc]https://openid.net/connect/ (OIDC), an identity layer built on top of OAuth2. OIDC is supported by most identity providers, including:
```

Comment:
This seems wrong, and the link already worked fine afaict.

Reply:
Yes, I was sure I removed it. fixed

Reply:
I'm still seeing this in the most recent version of the PR